rotkehlchen/tests/exchanges/test_woo.py

```diff
@@ -0,0 +1,137 @@
+from unittest.mock import call, patch
+
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.constants.assets import A_BTC, A_ETH, A_WOO
+from rotkehlchen.exchanges.data_structures import Trade, TradeType
+from rotkehlchen.exchanges.woo import API_MAX_LIMIT, Woo
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import Location, Timestamp
+
+
+def test_name():
+    exchange = Woo('woo', 'a', b'a', object(), object())
+    assert exchange.location == Location.WOO
+    assert exchange.name == 'woo'
+
+
+@pytest.mark.parametrize(('start_ts', 'end_ts'), [(0, 1), (1634600000, 1634610000)])
+def test_query_online_trade_history_basic(mock_woo, start_ts, end_ts):
+    mock_trades = [
+        {
+            'id': 1,
+            'symbol': 'SPOT_BTC_USDT',
+            'order_id': 101,
+            'executed_price': 50000.0,
+            'executed_quantity': 1.0,
+            'side': 'BUY',
+            'fee': 0.1,
+            'fee_asset': 'USDT',
+            'executed_timestamp': '1634600000.0',
+        },
+        {
+            'id': 2,
+            'symbol': 'SPOT_ETH_USDT',
+            'order_id': 102,
+            'executed_price': 3000.0,
+            'executed_quantity': 2.0,
+            'side': 'SELL',
+            'fee': 0.2,
+            'fee_asset': 'USDT',
+            'executed_timestamp': '1634610000.0',
+        },
+    ]
+    with patch.object(Woo, 'query_online_trade_history', return_value=mock_trades):
+        result = mock_woo.query_online_trade_history(
+            start_ts=Timestamp(start_ts),
+            end_ts=Timestamp(end_ts),
+        )
+    assert result == mock_trades
```

Comment:
I'm not sure what is being tested here, as you have it is a no-op since ti should return a list of trades and a tiemstamp range